packages/docs/src/styles/variables.css

```
@@ -14,8 +14,19 @@
   --accent-color: #dc6843;
   --logo-text-filter: none;
   --apple-logo-filter: invert(1);
+  --meta-logo-filter: none;
   --iapkit-text-color: #ffffff;
 
+  /* Typography */
+  --font-family-display: 'Manrope', 'Inter', sans-serif;
+  --font-family-sans:
+    'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)